Resources/i18n/ar.json

```diff
@@ -1,5 +1,6 @@
 {
-	"language":"English",
-	"stars":"Stars",
-	"test":"meow"
+  "font":"fonts/AndikaLowerCase-Regular_5dp.ttf",
```

Comment:
@protito Nice solution! In the future it will be necessary to use a different font for some of the languages, so this keeps the software scalable 👍